ratatui-widgets/src/paragraph.rs

```diff
@@ -151,11 +154,47 @@ impl<'a> Paragraph<'a> {
     where
         T: Into<Text<'a>>,
     {
+        Self::from_cow_text(Cow::Owned(text.into()))
+    }
+
+    /// Creates a new [`Paragraph`] widget with the given text.
+    ///
+    /// By default, the text is styled with [`Style::default()`], not wrapped, and aligned to the
+    /// left.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use ratatui::style::{Style, Stylize};
+    /// use ratatui::text::{Line, Text};
+    /// use ratatui::widgets::Paragraph;
+    /// let text = Text::raw("Hello, world!");
+    /// let paragraph = Paragraph::from_cow_text(Cow::Borrowed(&text));
+    /// ```
+    pub const fn from_borrowed_text(text: &'a Text<'a>) -> Self {
+        Self::from_cow_text(Cow::Borrowed(text))
+    }
+
+    /// Creates a new [`Paragraph`] widget with the given text.
+    ///
+    /// By default, the text is styled with [`Style::default()`], not wrapped, and aligned to the
+    /// left.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use ratatui::style::{Style, Stylize};
+    /// use ratatui::text::{Line, Text};
+    /// use ratatui::widgets::Paragraph;
+    /// let text = Text::raw("Hello, world!");
+    /// let paragraph = Paragraph::from_cow_text(Cow::Borrowed(&text));
+    /// ```
+    pub const fn from_cow_text(text: Cow<'a, Text<'a>>) -> Self {
```

Comment:
Same here - could this reasonably be `impl<'a> From<Cow<''a, Text<'a>> for Paragraph<'a>`